gapis/server/server.go

```diff
@@ -613,7 +626,8 @@ func (s *server) Status(
 			return nil
 		}
 	}
-	return stop, nil
+	<-ctx.Done()
+	return nil
```

Comment:
`stop` is no longer used? can it just be dropped?

Reply:
Good catch thanks!